portfolio/src/main/webapp/index.html

```diff
@@ -199,9 +199,9 @@ <h4 class="subheading">Global Movement</h4>
               </div>
               <div class="timeline-body">
                 <p class="text-muted">
-                  A video showing a police officer kneeling on George Floyd's neck goes viral following his death. The
-                  Black Lives Matter movement inspires protests to be organised all around the world opposing racial
-                  oppression.
+                  A video showing a police officer kneeling on George Floyd's neck went viral following his death. The
+                  Black Lives Matter movement inspired protests to be organised all around the world opposing racial
```

Comment:
Black Lives Matter --> BLM to be consistent

Reply:
It's the last I mention it, so I want to use the full name.